src/client/app/effects/github.ts

```diff
@@ -146,7 +147,7 @@ export class GitHubEffects {
             if (exception instanceof PlaygroundError) {
                 try {
                     let message = JSON.parse(exception.message);
-                    if (message.hasOwnProperty('type') && message.type === 'UpdateGistFailed') {
+                    if (message.hasOwnProperty('type') && message.type === GitHub.GitHubActionTypes.UPDATE_GIST) {
```

Comment:
Is hasOwnProperty check really necessary? I would think you could just use if (message.type === GitHub.GitHubActionTypes.UPDATE_GIST)